### PR TITLE
Adding the python script for image editor and the readme.md file

### PR DESCRIPTION
**Pull request by asmita20aggarwal**
# Description

This is my first pull request. I have added the python script for the image editor along with the readme.md file.

## Fixes: #647

Replace `647` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

when program will be executed this will show up :

WhatsApp Image 2021-03-23 at 12 49 44 AM

User will be able to save his file at the desired location :

WhatsApp Image 2021-03-23 at 12 54 43 AM

Original Image :

flower

Edited image (mystic meadows filter applied)

edited_flowers

### CODE REVIEW

File: Image editor python script/Image_editing_script.py [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+this function will ask the user if he wants a
+preview of edited image or not and will provide the same
+'''
+
+
+def edited_image_preview(image_name):
+
+    print("=======Do you want to preview the edited image ?=======")
+
+    preview_choice = input("Type 1 for yes and 2 for no : ")
```
Comment (kaustubhgupta) [inline Image editor python script/Image_editing_script.py:114]: Be consistent with upper and lower case:
```suggestion
    preview_choice = input("Type 1 for YES and 2 for NO : ")
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Minor changes needed:

**Comment by asmita20aggarwal** [comment]
@kaustubhgupta Sir, I have made the suggested changes.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Kushal997-das

**Review by santushtisharma10 — APPROVED** [review]
Nice @asmita20aggarwal 💯💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Comment by asmita20aggarwal** [comment]
Thankyou :)

**Pull request merged**